src/Models/Audit.php

```diff
@@ -21,6 +21,13 @@ class Audit extends Model implements \OwenIt\Auditing\Contracts\Audit
      */
     protected $guarded = [];
 
+    /**
+     * Is globally auditing disabled?
+     *
+     * @var bool
+     */
+    public static $auditingDisabled = false;
```

Comment:
What do you think about changing the property name to `$disableAuditingGlobally`? In version `13.6.6` our `Audit` entity broke because it's both using the `Auditable` trait (through a parent class) and the `Audit` trait. We use `public static $auditingDisabled = true;` from the `Auditable` trait to disable the Auditing of the `Audit` entity.

Now, that is fixed in this pr as well, but to make the code more readable renaming this property to `$disableAuditingGlobally` shows its intent more clearly imo

Reply:
No offence but it sounds like your model inheritance is a bit weird, perhaps a cleaner and safer fix would be if the library detected that the model was both an instanceof Auditable and an instanceof Audit and avoided creating some kind of infinite loop on its own :sweat_smile:

Reply:
It was merely a suggestion to rename the property for globally disabling auditing to `$disableAuditingGlobally` for clarity. Regardless of our inheritance tree.

Reply:
Although a check in the library to prevent infinite loops might be a nice addition as well :D 
Not immediately necessary for us since we fixed that on our side :)